tutorials/embedding/word2vec.py

```diff
@@ -42,7 +42,7 @@
 import numpy as np
 import tensorflow as tf
 
-import gen_word2vec as word2vec
+word2vec = tf.load_op_library(os.path.dirname(os.path.realpath(__file__)) + '/word2vec_ops.so')
```

Comment:
For some reason when I tried `tf.load_op_library('word2vec_ops.so'), I got this error: tensorflow.python.framework.errors_impl.NotFoundError: word2vec_ops.so: cannot open shared object file: No such file or directory.